jossPaper/joss.bib

```diff
@@ -113,5 +113,27 @@ @article{diease2011
 	doi={10.1038/jp.2010.172},
 	Volume =31,
 	Year = 2011}
+    
+@article{delmas2019analysing,
+  title={Analysing ecological networks of species interactions},
+  author={Delmas, Eva and Besson, Mathilde and Brice, Marie-H{\'e}l{\`e}ne and Burkle, Laura A and Dalla Riva, Giulio V and Fortin, Marie-Jos{\'e}e and Gravel, Dominique and Guimar{\~a}es Jr, Paulo R and Hembry, David H and Newman, Erica A and others},
+  journal={Biological Reviews},
+  volume={94},
+  number={1},
+  pages={16--36},
+  year={2019},
+  publisher={Wiley Online Library}
+}
+
+@article{vine2006google,
```

Comment:
Hi @Zakir-Hossine, please just cite https://scholar.google.com.au/. This citation is not appropriate.